lib/sim800.py

```diff
@@ -0,0 +1,809 @@
+"""SIM800 library for the TiLDA MK4"""
+
+___license___ = "MIT"
+___dependencies___ = []
+
+import machine
+import time
+import micropython
+
+uart_port = 1
+uart_default_baud = 115200
+uart_timeout = 28
+default_response_timeout = 2000
+
+status_pin = machine.Pin(machine.Pin.GPIO_SIM_STATUS, machine.Pin.IN)
+ringer_pin = machine.Pin(machine.Pin.GPIO_SIM_RI, machine.Pin.IN)
+pwr_key_pin = machine.Pin(machine.Pin.GPIO_SIM_PWR_KEY, machine.Pin.OUT)
+
+# Open the UART
+uart = machine.UART(uart_port, uart_default_baud, mode=machine.UART.BINARY, timeout=uart_timeout)
+dirtybuffer = False # Flag if the buffer could have residual end of reresponsesponces line in it?
+
+# A list of callback functions
+callbacks = []
+
+# Globals for remembering callback data
+clip = ""
+btpairing = ''
+
+# Check if the SIM800 is powered up
+def ison():
```

Comment:
[style, don't worry if it's too much effort] The rest of the lib functions are snake_cased

Reply:
nm, I've just realised its going to be in the booklet like that so it's probably best to leave it consistent within this library